src/main/java/app/commands/ShowAllContactsCommand.java

```diff
@@ -10,8 +11,12 @@
 public class ShowAllContactsCommand implements ActionCommand {
     @Override
     public String execute(HttpServletRequest request, HttpServletResponse response) {
-        List<Contact> contacts = ContactDaoImpl.getInstance().findAll();
-        request.setAttribute("contacts", contacts);
+        try {
+            List<Contact> contacts = ContactDaoImpl.getInstance().findAll();
+            request.setAttribute("contacts", contacts);
+        } catch (AppException ex) {
+            request.setAttribute("error", ex.getMessage());
+        }
         return "/views/contactList.jsp";
```

Comment:
This kind of variables should be set via constants, not hard-coding them. If somebody decides to change folder where your JSPs live it will be painful.